### PR TITLE
[TECH-2727] - Set up CI/CD for backend

### PR DESCRIPTION
**Pull request by ebram-tw**
This is a CI/CD pipeline to build the backend.
Instead of building the jar locally, this first pass is just taking the latest `jar` release from the main project.

### CODE REVIEW

File: tools/docker/openems-backend/Dockerfile
Comment (gsvitak) [inline tools/docker/openems-backend/Dockerfile:1]: please remember this repo is a public fork. Do we want to have TWI specific Dockefile so we can pull latest from public upstream

File: .circleci/config.yml
Comment (gsvitak) [inline .circleci/config.yml:1]: same comment. Please remember this repo is a public fork.

**Review by gsvitak — COMMENTED** [review]
Please see some of my comments. I will leave it to you to decide how we should proceed. Will defer to you perspective.

File: .circleci/config.yml
Comment (ebram-tw) [reply .circleci/config.yml:1]: I realize that, is there a concern you have here?

Comment (gsvitak) [reply .circleci/config.yml:1]: not big concern. just wanted to make sure you were aware.  Defer to you on the proper approach

**Pull request merged**